moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -63,6 +63,15 @@ class PlanningRequestAdapter
   {
   }
 
+  /// Initialize parameters using the passed NodeHandle
+  // TODO - Make initialize() a pure virtual function
```

Comment:
A deadline is missing:
```suggestion
  // TODO - Make initialize() a pure virtual function in O-turtle
```